improver_tests/blending/spatial_weights/test_SpatiallyVaryingWeightsFromMask.py

```diff
@@ -156,7 +155,7 @@ def test_aux_blending_coord(self):
            an AuxCoord"""
         expected = self.cube_to_collapse.copy()[:, 0, :, :]
         plugin = SpatiallyVaryingWeightsFromMask("forecast_period")
-        result = self.plugin._create_template_slice(self.cube_to_collapse)
+        result = plugin._create_template_slice(self.cube_to_collapse)
```

Comment:
(This is a change in behaviour, but I think it is a fix)

Reply:
FYI @MoseleyS and @fionaRust - if you could confirm this particular change is OK it would be great